UnityProject/Assets/Scripts/Player/Huggable.cs

```diff
@@ -104,7 +104,7 @@ private bool TryFieryHug()
 
 	private void Judgement(LivingHealthMasterBase puller)
 	{
-		if(!Cooldowns.TryStart(interaction.PerformerPlayerScript, this, NetworkSide.Server)) lastPuller = null;
+		if(Cooldowns.TryStart(interaction.PerformerPlayerScript, this, NetworkSide.Server) == false) lastPuller = null;
```

Comment:
break the body into a new line, it ugly enough already by bing this long of a condition and having ``== false``

Reply:
like this?